### PR TITLE
Fix/3337 refresh gallery preview when a content is modified

### PR DESCRIPTION
**Pull request by metzgegu**
READY TO REVIEW BUT NOT READY TO MERGE

#3337

## Checkpoints

These points must be checked before merging. Please don't edit them out.

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_app_gallery/src/container/Gallery.jsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ export class Gallery extends React.Component {
 
   liveMessageNotRelevant (data, state) {
     return Number(data.content.workspace_id) !== Number(state.config.appConfig.workspaceId) ||
-    Number(data.content.parent_id) !== Number(state.folderId)
+    (state.folderId !== undefined && Number(data.content.parent_id) !== Number(state.folderId))
```
Comment (gbonaspetti) [inline frontend_app_gallery/src/container/Gallery.jsx:98]: The TLM is relevant if `Number(data.content.parent_id) !== Number(state.folderId)` ?

Comment (metzgegu) [reply frontend_app_gallery/src/container/Gallery.jsx:98]: The TLM is revelent if `Number(data.content.parent_id) === Number(state.folderId)`*

Comment (gbonaspetti) [reply frontend_app_gallery/src/container/Gallery.jsx:98]: ok, my bad, I forgot the `Not` in the function name. PR approuved.

**Comment by gbonaspetti** [comment]
Tested on branch.

**Pull request merged**